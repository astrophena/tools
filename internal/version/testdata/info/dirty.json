{
  "name": "starlet",
  "version": "git",
  "commit": "f57b584d",
  "dirty": true,
  "built_at": "2024-06-29T16:46:18Z",
  "go": "go1.22.3",
  "os": "android",
  "arch": "arm64"
}
