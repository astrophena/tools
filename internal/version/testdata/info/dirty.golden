starlet git-f57b584d-dirty
built with go1.22.3, android/arm64
built at 2024-06-29T16:46:18Z
